imports/ofx/account.go

```diff
@@ -0,0 +1,58 @@
+package ofx
+
+import (
+	"github.com/aclindsa/xml"
+	"gorm.io/gorm"
+)
+
+type Account struct {
+	gorm.Model
+	Currency     string
+	BankID       string
+	AccountID    string
+	Type         string
+	Transactions []Transaction
+}
+
+func (Account) TableName() string {
+	return tablePrefix + "accounts"
+}
+
+func (a *Account) UnmarshalXML(decoder *xml.Decoder, start xml.StartElement) error {
+	type Alias Account
+
+	var data struct {
+		Alias
+		AccountInfo struct {
+			Currency string `xml:"CURDEF"`
+			Meta     struct {
+				BankID    string `xml:"BANKID"`
+				AccountID string `xml:"ACCTID"`
+				Type      string `xml:"ACCTTYPE"`
+			} `xml:"BANKACCTFROM"`
+			TransactionsStruct struct {
+				Transactions []Transaction `xml:"STMTTRN"`
+			} `xml:"BANKTRANLIST"`
+		} `xml:"STMTRS"`
+	}
+
+	if err := decoder.DecodeElement(&data, &start); err != nil {
+		return err
+	}
+
+	*a = Account(data.Alias)
+	a.Currency = data.AccountInfo.Currency
+	a.BankID = data.AccountInfo.Meta.BankID
+	a.AccountID = data.AccountInfo.Meta.AccountID
+	a.Type = data.AccountInfo.Meta.Type
+	a.Transactions = append(a.Transactions, data.AccountInfo.TransactionsStruct.Transactions...)
```

Comment:
I see that all other fields are reassigned and only this one changed with respect to the previous state. Is that ok?
```suggestion
	a.Transactions = data.AccountInfo.TransactionsStruct.Transactions
```

Reply:
nope, my mistake.